### PR TITLE
Code gardening: improved error messages

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**

### CODE REVIEW

File: exec_test.go [hunk 1]
```diff
@@ -194,7 +194,7 @@ func TestCommandStartWait(t *testing.T) {
 	testCases := map[string]testCase{
 		// Background context
 		"success":                     {distro: &realDistro, cmd: "exit 0"},
-		"failure fake distro":         {distro: &fakeDistro, cmd: "exit 0", wantErrOn: AfterWait},
```
Comment (didrocks) [inline exec_test.go:197]: TODO (in another PR maybe), subtests can be read as sentences when being more than one word or so, like "error when blablablabla". I find it easier to read.

Not related to that PR, but anther idea for code garderning!

**Review by didrocks — APPROVED** [review]
I like this kind of small adjustement which makes it way more readable (and you sneaked another fix in between, but I see you :p).

Approved!

File: exec_test.go (hunk 1 above)
Comment (EduardGomezEscandell) [reply exec_test.go:197]: 🌱🪴🌲

**Comment by EduardGomezEscandell** [comment]
> and you sneaked another fix in between

Hey, it's error-related :P

**Pull request merged**